src/Api/WebClientInterface.php

```diff
@@ -78,7 +79,7 @@ public function getLabels(array $shipmentIds): ?string;
 
     public function getAuthorizedHeaderWithContentType(): array;
 
-    public function createShipment(ShipmentInterface $shipment): array;
+    public function createShipment(ShipmentInterface $shipment, ?ShippingExportInterface $shippingExport = null): array;
```

Comment:
Why there can be null?

Reply:
Changed 👍 
I took too timid an approach.